region/brinstar/green/Early Supers Room.json

```diff
@@ -496,6 +496,31 @@
         "Note that if Samus takes more than 35 seconds after entering the room, before shooting the ceiling, the Waver will not come down."
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Morph",
+        {"refill": ["Energy"]},
+        "canSlowShortCharge",
+        {"canShineCharge": {"usedTiles": 30, "openEnd": 1}},
```

Comment:
`canSlowShortCharge` with 30 tile runway doesn't seem right...
canSlowShortCharge is releasing run as you turn blue to continue running.  And is being phased out.

Reply:
A regular 1-tap shortcharge seems to work fine: hold forward and then start holding dash just a moment later.

Reply:
A single stutter is safe, and any level of 1-tap works.  So its not mechanically hard but is a strat you want to practice once or twice.  That means it needs more of a knowledge check tech and `canRModeSparkInterrupt` can work for that.